doc/api/wasi.md

```diff
@@ -11,9 +11,9 @@ specification. WASI gives sandboxed WebAssembly applications access to the
 underlying operating system via a collection of POSIX-like functions.
 
 ```js
-'use strict';
-const fs = require('fs');
```

Comment:
Maybe stay commonjs for a while.

Reply:
Top-level await only works for ESM… Would you prefer if there were two examples: one ESM with TLA, and one CJS?

Reply:
I am good with esm only. cc @cjihrig

Reply:
Let's stick with just one version.